storage/src/mmr/bitmap.rs

```diff
@@ -77,19 +101,100 @@ impl<H: CHasher, const N: usize> Bitmap<H, N> {
             bitmap,
             next_bit: 0,
             mmr: Mmr::new(),
+            pruned_chunks: 0,
         }
     }
 
-    /// Return the number of bitmap chunks that have been pruned.
-    #[inline]
-    fn pruned_chunks(&self) -> usize {
-        self.mmr.pruned_to_pos() as usize
+    /// Restore a fully pruned bitmap from the metadata in the given partition.
```

Comment:
nit: This comment implies to me that you get the entire bitmap back. I like the wording in `write_pruned` that you still need to replay (but that this just preserves the hash of some variety).

Reply:
added that in.